main.cpp

```diff
@@ -19,15 +19,29 @@ using namespace clang::tooling;
 
 class CastCallBack : public MatchFinder::MatchCallback {
 public:
-    CastCallBack(Rewriter& rewriter) {
-        // Your code goes here
-    };
-
+    CastCallBack(Rewriter& rewriter): _rewriter(rewriter) {};

```

Comment:
```suggestion
    CastCallBack(Rewriter& rewriter): _rewriter(rewriter) {}
```
`;` is not needed here.